src/ServiceControl/Infrastructure/Settings/Settings.cs

```diff
@@ -205,6 +207,10 @@ public int ExpirationProcessBatchSize
 
         public bool ExposeApi { get; set; } = true;
 
+        public bool SqlStorageEnabled { get; set; }
```

Comment:
I feel this should be an `enum` as we will have more than just SqlStorage (for now there'll be Raven, Sql and InMemory)